metals/src/main/scala/scala/meta/internal/metals/codeactions/ExtractRenameMember.scala

```diff
@@ -45,11 +45,16 @@ class ExtractRenameMember(
         val fileName = path.filename.replaceAll("\\.scala$", "")
 
         val definitions = membersDefinitions(tree)
-        val sldNames = sealedNames(tree)
+        val sealedNames: List[String] = getSealedNames(tree)
         val defnAtCursor =
-          definitions.find(_.name.pos.toLSP.overlapsWith(range))
-
-        def canRenameDefn(defn: Member): Boolean = {
+          definitions.find(_.member.name.pos.toLSP.overlapsWith(range))
+
+        def canRenameDefn(
+            defn: Member,
+            fileName: String,
+            defnAtCursor: Option[Member],
+            uri: String
+        ): Boolean = {
```

Comment:
```suggestion
        def canRenameDefn(defn: Member): Boolean = {
```

The rest is defined.